### ISSUE TITLE
Hide broken radio stations

### ISSUE THREAD

**Issue by davidhedlund** [segler-alex/radiobrowser#33]
Can you please add an option with a checkbox "Hide broken stations" next to the search field that is checked by default to the site?

Broken stations should only be visible in http://www.radio-browser.info/gui/#/broken, nobody want to have broken URLs in their playlists because that will for example popup an error message in VLC who won't playing the next URL before you remove the popup. This is extremely annoying.

**Comment by davidhedlund** [comment]
@segler-alex This should be regarded as a high priority issue to fix: Its very easy to implement and it gives the users a very good enhancement.

**Comment by davidhedlund** [comment]
I get the idea that the broken stations are displayed when you surf radio-droid.org with a web browser so you can see whats broken and edit and repair them. But the broken stations should be excludes in the saved playlists.

**Comment by davidhedlund** [comment]
@segler-alex Can you please have a new look on this?

**Comment by davidhedlund** [comment]
@segler-alex Can you please make a small effort to fix this issue?

Its still extremely annoying.

**Comment by 0ttoman** [comment]
You could tell VLC to skip the link if it doesn't work. This is not a HIGH PRIORITY nor URGENT or Important..
There are streams that only go online at certain times thus radio-browser has the broken tag. You would exclude those 'daylight only' streams.
If the stream wasn't online for 60 days it is marked deleted. Deleted streams are excluded from the playlist.

**Comment by davidhedlund** [comment]
@0ttoman That make sense, I think this should be documented in the FAQ though.

**Comment by gkaklas** [comment]
I agree that this is not a high priority issue. This would probably be an interesting feature, but as @0ttoman said, excluding stations that have temporary connectivity problems or aren't constantly online would be a problem.

I think this can be closed... :thinking:

**Comment by davidhedlund** [comment]
> I think this can be closed... thinking

I disagree.
